src/test/fuzz/string.cpp

```diff
@@ -78,7 +78,9 @@ FUZZ_TARGET(string)
     }
     (void)SanitizeString(random_string_1);
     (void)SanitizeString(random_string_1, fuzzed_data_provider.ConsumeIntegralInRange<int>(0, 3));
+    #ifndef WIN32
     (void)ShellEscape(random_string_1);
+    #endif
```

Comment:
I don't really like changing code in a build change. Maybe exclude the failing ci config via `--disable-fuzz` for now?